### ISSUE TITLE
buttons with underlines

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#1035]
see underline:

![image](https://cloud.githubusercontent.com/assets/2367265/23730771/3c227ce2-041e-11e7-9fe0-7e060508fb0c.png)

This happens in other places as well, e.g. when adding users to classes:

![image](https://cloud.githubusercontent.com/assets/2367265/24275597/c77ed4c2-0fed-11e7-995b-27571caf0e79.png)

We should look holistically at why this problem happens and prevent it.

**Comment by christianmemije** [comment]
This occurs when we wrap a button with a link.

**Comment by christianmemije** [comment]
Would be cool if the `icon-button` had a `link` prop or if we had a new `button-link` component since wrapping a button in a link is a very common pattern.

**Comment by indirectlylit** [comment]
> wrapping a button in a link is a very common pattern

We should be wrapping buttons in links ... it should be just a link or a button, not both.

I do agree that having a `button-link` (maybe `link-button`?) component would be the right solution.

**Comment by radinamatic** [comment]
> ...it should be just a link or a button, not both.

Yes!!! :+1: 

And decision about one or the other should be based on the actual function/goal that the element is supposed to perform. Some good reads about `Links vs Buttons` from an accessibility point of view:

## [Links are not buttons. Neither are DIVs and SPANs](http://www.karlgroves.com/2013/05/14/links-are-not-buttons-neither-are-divs-and-spans/)

### TL;DR

> DIVs and SPANs are not buttons
> If it navigates, it is a link. Use link markup with a valid hypertext reference
> If it triggers an action, it is a button. Use a BUTTON element
> If you don’t like how a button looks, style it with CSS.

## [Links, Buttons, Submits, and Divs, Oh Hell](http://adrianroselli.com/2016/01/links-buttons-submits-and-divs-oh-hell.html)

### TL;DR
https://twitter.com/aardrian/status/734792836667805696

## [Links vs. Buttons in Modern Web Applications](https://marcysutton.com/links-vs-buttons-in-modern-web-applications/)

### Why is it important?
> If a screen reader user calls tech support and gets instructions to “click the button” in your UI that’s really coded as a link, they may have trouble finding it. Also, consider voice interfaces: if you say a command to click a button but it’s really coded as a link, you might have problems, no?

**Comment by indirectlylit** [comment]
refs #1987

**Comment by christianmemije** [comment]
Fixed in https://github.com/learningequality/kolibri/pull/2255 :man_dancing:
